### ISSUE TITLE
Submit to CRAN

### ISSUE THREAD

**Issue by rguliev** [r-hyperspec/unmixR#13]
In GitLab by @bryanhanson on Jun 3, 2016, 03:24

By the end of the summer, we should be able to submit `unmixR` to CRAN.

**Comment by rguliev** [comment]
In GitLab by @bryanhanson on Aug 7, 2017, 15:16

Here are the steps to get (almost) ready for CRAN:
1. Move to new branch CRAN
2. Remove large files (from testing) in folder inst
3. Move testthat instances to the typical place so hacked functions are not needed for checking (which can't pass CRAN checks). See #44 
4. Add [nojss] to vignette for the time being -- removes JSS header/footers
5. See what's left in terms of check issues.
6. Also, run the package by win-builder and rhub to check multiple platforms.

Let me know if you want me to proceed.  It's easy enough to undo #44 but adding that back will be tough once branch CRAN is merged back into devel_2017 or master (but we have to pass CRAN checks somehow).
